### ISSUE TITLE
resolveConventionName(...) is not a function

### ISSUE THREAD

**Issue by AndreiBacescu** [dotansimha/graphql-code-generator#1656]
**Describe the bug**
 
I'm trying to migrate to version  1.0.6 but when I try to add *namingConvention: enumValues: change-case\#lowerCase * parameter in `codegen.yml` I got this error.

```
  √ Parse configuration
  > Generate outputs
    > Generate src/generated/graphql.ts
      √ Load GraphQL schemas
      √ Load GraphQL documents
      × Generate
        → resolveConventionName(...) is not a function
````

**To Reproduce**

1. My `codegen.yml` config file:

```
overwrite: true
schema: "http://localhost:4000/graphql"
documents: null
generates:
  src/generated/graphql.ts:
    config: 
      namingConvention:
        enumValues: change-case#lowerCase
      contextType: ../graphql-utils/types#GlobalContext
      mappers:
        User: ./prisma-client/index#User
        Value: ./prisma-client/index#Value
    plugins:
      - "typescript"
      - "typescript-resolvers"
```

**Environment:**

- OS:  Windows 10 Pro 
- NodeJS: v11.9.0

**Comment by dotansimha** [comment]
Thank you for reporting it, @AndreiBacescu. 
I managed to reproduce and fix it in this PR: https://github.com/dotansimha/graphql-code-generator/pull/1663

**Comment by dotansimha** [comment]
Fixed in 1.0.7 :)
